docs/root/configuration/security/_include/sds-source-example.yaml

```diff
@@ -0,0 +1,107 @@
+node:
+  cluster: envoy_cluster
+  id: envoy_node
+
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+        socket_address:
+          address: 0.0.0.0
+          port_value: 8000
+    filter_chains:
+    - transport_socket:
+        name: envoy.transport_sockets.tls
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.DownstreamTlsContext
+          common_tls_context:
+            tls_certificate_sds_secret_configs:
+            - name: server_cert
+              sds_config:
+                resource_api_version: V3
+                api_config_source:
+                  api_type: GRPC
+                  transport_api_version: V3
+                  grpc_services:
+                  - envoy_grpc:
+                      cluster_name: sds_server_mtls
+            validation_context_sds_secret_config:
+              name: validation_context
+              sds_config:
+                resource_api_version: V3
+                api_config_source:
+                  api_type: GRPC
+                  transport_api_version: V3
+                  grpc_services:
+                  - envoy_grpc:
+                      cluster_name: sds_server_uds
+  clusters:
+  - name: sds_server_mtls
+    typed_extension_protocol_options:
+      envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+        "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+        explicit_http_config:
+          http2_protocol_options:
+            connection_keepalive:
+              interval: 30s
+              timeout: 5s
+    load_assignment:
+      cluster_name: sds_server_mtls
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              socket_address:
+                address: 127.0.0.1
+                port_value: 8234
+    transport_socket:
+      name: envoy.transport_sockets.tls
+      typed_config:
+        "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+        common_tls_context:
+          tls_certificates:
+          - certificate_chain:
+              filename: certs/servercert.pem
+            private_key:
+              filename: certs/serverkey.pem
+  - name: sds_server_uds
+    typed_extension_protocol_options:
+      envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+        "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+        explicit_http_config:
+          http2_protocol_options: {}
+    load_assignment:
+      cluster_name: sds_server_uds
+      endpoints:
+      - lb_endpoints:
+        - endpoint:
+            address:
+              pipe:
+                path: /tmp/uds_path
+  - name: example_cluster
+    connect_timeout: 0.25s
```

Comment:
`connect_timeout` is definitely not required (it used to be but has a default now)

Reply:
This was present in existing example, I felt this has some value being kept here for users to know this parameter. Other clusters don't use the same, indicating its okay to omit. Wdyt?

Reply:
i think its not necessary - its only present because there didnt used to be a default - when it was added, i removed from (most) other configs

Reply:
ack, thanks for sharing this